website/content/docs/configuration/seal/seal-ha.mdx

```diff
@@ -0,0 +1,97 @@
+---
+layout: docs
+page_title: Seal High Availability (Beta) - Seals - Configuration
+description: |-
+  How to configure multiple Seals for high availability.
+---
+
+# Seal High Availability (Beta)
+
+<Note title="Seal HA functionality requires Vault Enterprise">
+
+  Seal High Availability requires Vault Enterprise.
+
+</Note>
+
+-> **Warning:** This feature is available as a Beta for evaluation and should not 
+be used in production deployments of Vault.
+
+[Seal High Availability](/vault/docs/content/concepts/seal#Seal-High-Availability) 
+is the ability to configure more than one seal in order to have resilience against
+outage of a seal service or mechanism.   
+
+Using Seal HA involves configuring extra seals in Vault's server configuration file
+and restarting Vault, after having enabled the Seal HA beta feature by setting
+the environment variable:
+
+`VAULT_ENABLE_SEAL_HA_BETA=true`
+
+Before using Seal HA, one must have upgraded to Vault 1.15 or higher, and started
```

Comment:
Another instance here I believe of talking about the migration needing to occur without the flag set.